torchaudio/functional/filtering.py

```diff
@@ -887,11 +870,43 @@ def lfilter(
 
     output = padded_output_waveform[:, n_order - 1:]
 
-    if clamp:
-        output = torch.clamp(output, min=-1.0, max=1.0)
-
     # unpack batch
     output = output.reshape(shape[:-1] + output.shape[-1:])
```

Comment:
It seems that this line is missing from the new ```lfilter``` and I don't find it in the C++ code. Did you mean to port it?

Reply:
This line can be found at L908, I don't think it's necessary to port it into c++ for autograd support.

Reply:
On L908 I see torch.clamp, but what about the unpacking of the batch, i.e.
```
output = output.reshape(shape[:-1] + output.shape[-1:])
```
. 

```_lfilter_core``` does the reshape before returning (unpacking) but lfilter_core in lfilter.cpp does not.

Reply:
`lfilter_core` in cpp did reshaping at the return statement
```c++
  return output.view(shape);
```
I should use `reshape` instead of `view`, sorry for confusion, will fix it soon.